apps/console/src/protected-app.tsx

```diff
@@ -151,18 +154,23 @@ export const ProtectedApp: FunctionComponent<AppPropsInterface> = (): ReactEleme
                 .get(Config.getServiceResourceEndpoints().wellKnown)
                 .then((response: AxiosResponse) => {
                     // Use token endpoint to extract the host url.
-                    const splitted: string[] = response.data.token_endpoint?.split("/") ?? [];
+                    const splitted: string[] = response?.data?.token_endpoint?.split("/") ?? [];
                     const serverHost: string = splitted.slice(0, -2).join("/");
-                    
+
                     window[ "AppUtils" ].updateCustomServerHost(serverHost);
+                })
+                .catch((error) => {
+                    // In case of failure customServerHost is set to the serverHost
```

Comment:
Add `.` at the end.

Reply:
Fixed with 8535eecb987fefadc2bc7058e3a9070e260baf10